app/models/server/models/_Base.d.ts

```diff
@@ -0,0 +1,19 @@
+export declare class Base {
+	model: any;
+
+	constructor(nameOrModel: string);
+
+	tryEnsureIndex(...args: any[]): any;
```

Comment:
Could define better the typing here, and, at least, the return of each other method